library/core/src/primitive.rs

```diff
@@ -25,10 +25,10 @@
 //! pub struct bool;
 //!
 //! impl QueryId for bool {
-//!     const SOME_PROPERTY: core::primitive::bool = true;
+//!     const SOME_PROPERTY: ::core::primitive::bool = true; // We also used `::core` instead of `core`, because `core` can be shadowed, too
```

Comment:
This should probably mention what the practical difference between `core` and `::core` is? Briefly, I know it is explained at length somewhere else.

Reply:
@workingjubilee , i fixed